gdcdictionary/schemas/follow_up.yaml

```diff
@@ -1604,6 +1846,12 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
+  imaging_suv:
+    type: number
+
+  imaging_suv_max:
+    type: number
+
```

Comment:
qn. are there appropriate mins or maxes here? like maybe a min of 0?

Reply:
I read the description and I have no idea.  @tlicht3 do you know offhand?  I don't suggest we put too much research into this.

Reply:
@wwysoc2 and @wteouchicago I think we can ask EVS for this when we add the term to NCIt... after reading about it, I agree that it's unlikely to be negative, but I don't know enough about it to know for sure.